<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tupianDialog</class>
 <widget class="QDialog" name="tupianDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="xlineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="ylineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="tlineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>fade</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>big</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>move</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>no</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>t:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>效果:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
